components/core/marketing/CodeTerminal.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from "react";
+import * as Constants from "~/common/constants";
+import * as System from "~/components/system";
+
+import { css, keyframes } from "@emotion/react";
+
+const blinkCursor = keyframes`
+0% {
```

Comment:
formatting.

Reply:
Can you clarify this particular formatting example, when Keyframes are used elsewhere it matches this format, for Example DataPicker.js so I think I'm missing something?

`const expand = keyframes`
  0% {
    max-height: 0px;
    overflow: hidden;
  }
  100% {
    max-height: 400px;
    overflow: hidden;
  }
`;`

Reply:
Look at your spacing, look at the spacing in that example you copy and pasted